run_unittests.py

```diff
@@ -7507,6 +7507,20 @@ def test_pkg_config_option(self):
             '-Dbuild.pkg_config_path=' + os.path.join(testdir, 'build_extra_path'),
             '-Dpkg_config_path=' + os.path.join(testdir, 'host_extra_path'),
         ])
+    
+    def test_run_native_test(self):
+        '''
+        https://github.com/mesonbuild/meson/issues/7997
+        check run native test in crossbuild without exe wrapper
+        '''
+        testdir = os.path.join(self.unit_test_dir, '88 run native test')
+        stamp_file = os.path.join(self.builddir, 'native_test_has_run.stamp')
+        self.meson_cross_file = os.path.join(testdir, 'cross.txt')
```

Comment:
You should not need to have your own cross file. The `setUp` method of `LinuxCrossArmTests` automatically sets this to the common Arm cross compilation file. You should be able to remove this line and your custom cross file as well.

Other than that, looks good.

Reply:
fixed